src/internal/_stepCat.js

```diff
@@ -20,7 +22,7 @@ module.exports = (function() {
   var _stepCatObject = {
     '@@transducer/init': Object,
     '@@transducer/step': function(result, input) {
-      return merge(
+      return _objectAssign(
```

Comment:
> Prehistory as in Eweda?

Prehistory as in I'm too lazy to dig up when it changed.  :smiling_imp: 

> Oh boy http://goo.gl/2cFsoO

That's interesting.  We'd have to look to see if there's significant difference in API, but if not, it looks like there's some real optimization possible...